src/styles/styles.ts

```diff
@@ -1442,6 +1442,10 @@ const styles = (theme: ThemeColors) =>
             alignItems: 'center',
         },
 
+        rightLabelMenuItem: {
+            width: 65,
```

Comment:
I don't like setting absolute width here. What if text exceeds this width?

Reply:
Just tested. Really bug. No need to set width


![Simulator Screenshot - iPhone 15](https://github.com/Expensify/App/assets/108292595/11cf3b85-8824-4315-b6d8-effc6aaa3d64)